gui/autotests/tst_gui.cpp

```diff
@@ -0,0 +1,31 @@
+#include <QCoreApplication>
```

Comment:
Any reason not to use googletest here, like we do for controller tests?

Reply:
just cause it's a QT application

Reply:
I guess it makes sense if we want to unit-test any QT-specific stuff like slots and signals. But I would prefer if we used a consistent testing framework whenever possible, i.e. for non-QT-specific GUI code we should still use googletest. It might be possible to use it for QT code too.

But it's fine to leave that out of this PR.